jobs/build/pre-release/Jenkinsfile

```diff
@@ -132,7 +132,7 @@ node {
             def CLIENT_TYPE = "ocp-dev-preview"
 
             stage("validation") {
-                release.stageValidation(quay_url, dest_release_tag, -1, params.PERMIT_PAYLOAD_OVERWRITE)
+                release.stageValidation(quay_url, dest_release_tag, -1, params.PERMIT_PAYLOAD_OVERWRITE, params.PERMIT_ALL_ADVISORY_STATES, params.FROM_RELEASE_TAG, arch)
```

Comment:
I don't think we actually attach images to advisories for dev previews. @sosiouxme Can you confirm this?

Reply:
You're right, we do not, so you can't validate that part, or most of the rest of the things we might validate during a release... you do have to match the method signature though, I think that's all this does.